uber/config.py

```diff
@@ -111,6 +111,15 @@ def get_attendee_price(self, dt):
     def get_group_price(self, dt):
         return self.get_attendee_price(dt) - self.GROUP_DISCOUNT
 
+    def get_badge_count_by_type(self, badge_type):
```

Comment:
Semi-related side note since you mention our "everyexpanding ``__getattr__`` override: how do you feel about having to modify ``__getattr__`` every time to do this kind of stuff?  I'm cool with it, but it would also be fairly easy to implement a special type of property like ``derived_property`` to let us write code like

```python
@derived_property('BEFORE_')
def before(self, dt):
    return bool(dt and localized_now() > dt)

@derived_property('AFTER_')
def after(self, dt):
    return bool(dt and localized_now() < dt)

@derived_property('_COUNT')
def count(self, badge_type):
    with sa.Session() as session:
        return session.query(sa.Attendee).filter_by(badge_type=badge_type, status=c.COMPLETED_STATUS).count()
```

That kind of refactor is certainly out of scope for this PR, but I figured I'd check whether it's the general kind of thing you'd find appealing, or if you'd prefer to just keep adding elif clauses to ``__getattr__``.